packages/gasless-wallet/README.md

```diff
@@ -43,13 +43,13 @@ const gaslessWalletContractAddress = gaslessWallet.getAddress();
 
 ```typescript
 const isGaslessWalletAlreadyDeployed = await gaslessWallet.isDeployed();
-const isGaslessWalletAlreadyInitialized = gaslessWallet.isInitialized();
+const isGaslessWalletAlreadyInitialized = gaslessWallet.isInitiated();
```

Comment:
Should this also be renamed to `isGaslessWalletAlreadyInitiated`?

Reply:
Updated, thanks